ts/index.d.ts

```diff
@@ -112,6 +116,10 @@ interface LinkResponse {
   result?: LinkResult;
 }
 
+interface UpdateCurrentUserPasswordResult {
```

Comment:
Looks like you added `UpdatePasswordResponse` in favor of this so this can probably be removed

Reply:
Great catch!